contracts/interfaces/IHATVault.sol

```diff
@@ -73,6 +73,8 @@ interface IHATVault is IERC4626Upgradeable {
 
     /**
     * @notice Initialization parameters for the vault
+    * @param name The vault's name (concatinated as "HATs Vault " + name)
```

Comment:
```suggestion
    * @param name The vault's name (concatenated as "Hats Vault " + name)
```

Reply:
But it is "HATs Vault", do we want to change this capitalization?

Reply:
yes keep this, as we discussed :)